pos_deposit/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,13 @@
+To configure this module, you need to go to *Point of Sale > Products*.
+
+#. Select or add at least one 'deposit product', such as 'Bottle 1L',
+   with a non-zero price and with the checkbox 'Is Deposit' set to True.
+
+#. Select or add at least one product of which 'Use Deposit' is checked,
+   such as '1 liter of cola'. In the accompanying product selection field,
+   select the bottle. The two products are now linked.
+
+#. When someone brings back bottles to the store, of course you want to
+   refund him. For that purpose, create new products of negative price,
+   such as 'Return/refund bottle 1L'. These products don't need to be
```

Comment:
That's not the better to handle stock of deposited product.
Why not just refund the bottle. The stock will be ok. Don't you think ?

Reply:
True, but typically people bring this in as part of their next order, so the refund of several bottles would need to be simultaneous with the buying of new products. Is that something Odoo 13.0 supports natively to your knowledge? (Excuse my laziness to test it myself)

Reply:
I've tested and we can just sell the bottle with a negative quantity, I guess that's what you meant also. I've removed this comment from the README.

Reply:
indeed. But we can imagine that when scanning a product with deposit, it is looking for the according deposited product and make +1 on quantity. Then, if the customer returns some bottles, cashier can set -x on quantity. Do'nt you think ?

Reply:
Yes true! So I think this is handled.